src/test/java/connectify/model/person/PersonListTest.java

```diff
@@ -0,0 +1,42 @@
+package connectify.model.person;
+
+import static connectify.testutil.Assert.assertThrows;
+import static connectify.testutil.TypicalPersons.ALICE;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import connectify.testutil.PersonBuilder;
+import connectify.testutil.PersonListBuilder;
+
+public class PersonListTest {
```

Comment:
Very comprehensive use of test cases to ensure code coverage

Reply:
Thanks for the feedback! We will continue updating the test cases if appropriate when we make new changes.